go.mod

```diff
@@ -3,55 +3,54 @@ module github.com/grafana/agent
 go 1.15
 
 require (
```

Comment:
I wish I knew why sometimes `go mod vendor` changes the indentation depending on what computer I'm on 😞